### PR TITLE
add -w option for opening MDN page in browser

### PR DESCRIPTION
**Pull request by greg-js**
Great little project! I was actually just about to make something like this myself but you beat me to it and probably did it better than I would have :)

I wanted this feature to also just open the page in a browser from the command-line though. In particular, I wanted to be able to quickly look at something in the terminal, and then do `!! -w` to go straight to the page to read more, rather than copying the URL or searching for it again in my browser. Anyway, I made a quick stab at it for my own personal purposes. Not sure if it's something you want for your project but thought I'd submit the PR. Anyway, thanks for saving me some work by writing this!

### CODE REVIEW

**Comment by rafaelrinaldi** [comment]
@greg-js Hey Greg! Thanks for your contribution. Nice to see this little tool being useful to others too! It's something I've wanted for a while.

So, I really like your idea I was just going to ask you two things:
- Can we use `--open` instead of `--web`? I think it's more straightforward and other tools use the same convention too (express and BrowserSync).
- This one is optional but it would be nice to return a `Promise` on `fetch()` no matter what so consumers don't have to check manually.

Thanks again for taking the time to contribute!

File: index.js [hunk 1]
```diff
@@ -71,7 +72,7 @@ const format = (markup, url) => {
   console.log(`${chalk.dim(url)}`);
 };
 
-const fetch = (keyword, language) => {
+const fetch = (keyword, language, web) => {
```
Comment (rafaelrinaldi) [inline index.js]: @greg-js You can use `shouldOpen` here.

File: index.js [hunk 2]
```diff
@@ -79,6 +80,11 @@ const fetch = (keyword, language) => {
       'user-agent': 'https://github.com/rafaelrinaldi/mdn'
     }
   };
+
+  if (web) {
+    return open(url);
```
Comment (rafaelrinaldi) [inline index.js]: @greg-js Ideally `fetch()` always return a `Promise`. This is optional and can be addressed by another PR :+1:

**Comment by greg-js** [comment]
I wasn't completely sure about returning the promise, is this what you were thinking of? And yep, it's a fantastically useful little tool for sure, and your implementation is right on point!

**Comment by rafaelrinaldi** [comment]
@greg-js Leave at it is! It's awesome already. Thanks again for contributing!

**Pull request merged**